zerver/lib/actions.py

```diff
@@ -2126,19 +2126,26 @@ def notify_reaction_update(
     # Update the cached message since new reaction is added.
     update_to_dict_cache([message])
 
-    # Recipients for message update events, including reactions, are
-    # everyone who got the original message.  This means reactions
-    # won't live-update in preview narrows, but it's the right
-    # performance tradeoff, since otherwise we'd need to send all
-    # reactions to public stream messages to every browser for every
-    # client in the organization, which doesn't scale.
-    #
-    # However, to ensure that reactions do live-update for any user
-    # who has actually participated in reacting to a message, we add a
-    # "historical" UserMessage row for any user who reacts to message,
-    # subscribing them to future notifications.
-    ums = UserMessage.objects.filter(message=message.id)
-    send_event(user_profile.realm, event, [um.user_profile_id for um in ums])
+    # Recipients for stream message update events, including reactions, are
+    # everyone who is subscribed to the stream the message belongs to.
+    # Credit goes to Steve Howell who helped me creating this patch.
```

Comment:
This sort of credit belongs in the commit message, not the comments, since it's talking about the change, not the final code :).

Reply:
Yeah, I realized that as well but didn't consider changing the code in the actual commit which I should have done. Don't worry will fix the complete PR.